backend/src/main/scala/cromwell/backend/BackendJobExecutorActor.scala

```diff
@@ -0,0 +1,83 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendJobExecutorActor._
+import cromwell.backend.model.JobDescriptor
+
+object BackendJobExecutorActor {
+
+  // Commands
+  sealed trait BackendJobExecutorActorMessage
+  case object Prepare extends BackendJobExecutorActorMessage
+  case object Execute extends BackendJobExecutorActorMessage
+  case object Stop extends BackendJobExecutorActorMessage
+  case object CleanUp extends BackendJobExecutorActorMessage
+
```

Comment:
Should BackendJobExecutorActor notify for intermediate status of a task? Such as running, onHold, aborting?
If so, we need to define intermediate statuses events and work with subscriptions here.